README.md

```diff
@@ -0,0 +1,1091 @@
+# PHP Extended Coding Style Guide
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119][].
+
+[RFC 2119]: http://tools.ietf.org/html/rfc2119
+
+## Overview
+
+This specification extends, expands and replaces [PSR-2][], the coding style guide and
+requires adherence to [PSR-1][], the basic coding standard.
+
+Like [PSR-2][], the intent of this specification is to reduce cognitive friction when
+scanning code from different authors. It does so by enumerating a shared set of rules
+and expectations about how to format PHP code. This PSR seeks to provide a set way that
+coding style tools can implement, projects can declare adherence to and developers
+can easily relate to between different projects. When various authors collaborate
+across multiple projects, it helps to have one set of guidelines to be used among
+all those projects. Thus, the benefit of this guide is not in the rules themselves
+but the sharing of those rules.
+
+[PSR-2][] was accepted in 2012 and since then a number of changes have been made to PHP
+which has implications for coding style guidelines. Whilst [PSR-2] is very comprehensive
+of PHP functionality that existed at the time of writing, new functionality is very
+open to interpretation. This PSR, therefore, seeks to clarify the content of PSR-2 in
+a more modern context with new functionality available, and make the errata to PSR-2
+binding.
+
+### Previous language versions
+
+Throughout this document, any instructions MAY be ignored if they do not exist in versions
+of PHP supported by your project.
+
+### Example
+
+This example encompasses some of the rules below as a quick overview:
+
+~~~php
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor\Package;
+
+use Vendor\Package\{ClassA as A, ClassB, ClassC as C};
+use Vendor\Package\SomeNamespace\ClassD as D;
+
+use function Vendor\Package\{functionA, functionB, functionC};
+
+use const Vendor\Package\{ConstantA, ConstantB, ConstantC};
+
+class Foo extends Bar implements FooInterface
+{

```

Comment:
i believe we wanted to keep the braces on the same line to standardize code style between languages